src/Icons/Icons.tsx

```diff
@@ -779,3 +788,10 @@ const AlignRightIcon: React.FC<Props> = ({ size = 'large', fill = DEFAULT_ICON_C
   </IconWrapper>
 );
 export { AlignRightIcon, alignRightUrl };
+
+const GoToIcon: React.FC<Props> = ({ size = 'large', fill = DEFAULT_ICON_COLOR, ...props }) => (
```

Comment:
I think it's ok for now, but this is so close to "ARROW MARK" the we should talk to design about the ability for it just use the same icon. The answer may be that they are different, but it would help build a consistency, otherwise we could end up with 15 different versions of arrow icons , at which point there wouldn’t be much reason to have a library for them.

Reply:
That's a good point, I also felt like just using the arrow mark, however, I think here it's ok to have both as we use them for very different purposes, but in any case, I agree we don't need any more versions.